doc/source/parameters/cfd/simulation_control.rst

```diff
@@ -1,108 +1,170 @@
-    * ``bdf3`` (3rd order backward differentiation)
-    * ``sdirk2`` (2nd order singly diagonally implicit Runge Kutta)
-    * ``sdirk3`` (3rd order singly diagonally implicit Runge Kutta)
+* ``output control``: control for the output of the simulation results. The available options are: 
+	* ``iteration``: results will be outputted at constant iteration frequency
+	* ``time`` : results will be outputted at constant time
 
-* The ``bdf startup method`` parameter controls the scheme used to start a high order bdf scheme.
+* ``output frequency``: controls after which number of iterations the ``.pvd`` / ``.vtu`` results are written. This parameter is only used when ``set output control = iteration``.
 
-* The ``output frequency`` controls after which number of iterations the ``.pvd`` / ``.vtu`` results are written. If ``output frequency = 0``, no ``.pvd`` / ``.vtu`` file will be written.
+.. tip::
+	If ``set output frequency = 0``, no ``.pvd`` / ``.vtu`` file will be written.
 
-* The ``output name`` and ``output path`` control the name of the output files and their directory.
+	If the ``output frequency`` is set at a higher number than the total number of iterations in the simulation, the startup iteration will still be outputted.
 
-* The ``output boundaries`` controls if the boundaries of the domain are written to a file. This will write additional VTU files made of the contour of the domain. This is particularly useful for the visualisation of 3D flows with obstacles or objects.
 
-* The ``subdivision`` parameter enables sub-division of the mesh cells to enable visualisation of high-order elements with Paraview. Generally, we advise to use a subdivision level of (n-1) for interpolation order of n. For example, a Q2-Q2 interpolation could be visualized with ``subdivision=1``.
+* ``output time``: controls the time when the ``.pvd`` / ``.vtu`` results are written. This parameter is only used when ``set output control = time``.
 
-* The ``time end`` parameter indicates at which value of the time to end the simulations if they are transient.
+* ``group files``: number of ``.vtu`` files generated in a parallel simulation
 
-* The ``time step`` parameter controls the value of the time step.
+.. tip::
+	This parameter allows to reduce the number of files generated when the simulation is run with a large number of processors. ``set group files = 1`` ensures that a single ``.vtu`` file will be generated. In this case, the file is written using MPI IO functionalities.
 
-* The ``adapt`` parameter controls if adaptive time-stepping is enabled. If adaptive time-stepping is enabled, the time-step will evolve to ensure that the 'max cfl' value is reached.
+	The value for this parameter should always be a compromise between keeping a low number of files but preventing excessive MPI communications. We have found that the default value of 1 does not have a significant impact on performance on Compute Canada clusters.
```

Comment:
```suggestion
	The value for this parameter should always be a compromise between keeping a low number of files but preventing excessive MPI communications. We have found that the default value of 1 does not have a significant impact on performance on Compute Canada clusters. However, as soon as the size of the output vtu file reaches 1Gb, it is preferable  to start splitting them into multiple smaller files as this may lead to corrupted files on some file systems.
```

Reply:
relevant indeed! Thank you